chore(ci): add issue labeling reporter (#27833)
renovatebot/renovate: docs/development/issue-labeling.md

</details>

Apply the `self-hosted` label when an issue is applicable only to users who self-administer their own bot.

## Automated label report

> There is a GitHub Action to automate the labeling process of issues, the job runs once in a week and checks for issues without the proper labels.

Reviewer: ```suggestion
We have a GitHub Action that checks if we're using the correct labels on our issues.
The action runs once in a week and checks for issues without the proper labels.
```

> If it finds any issues missing labels, it creates a new issue with the missing labels and details the issues found so any collaborator can label them accordingly.

Reviewer: ```suggestion
If the action finds any issues missing labels, it creates a new issue with the missing labels and details the issues found so any collaborator can label them accordingly.
```
